### PR TITLE
Replace 'Windows PowerShell' with 'PowerShell' in resx files

### PR DESCRIPTION
**Pull request by iSazonov**
Address #3345 

## Motivation

After removing FullCLR from the repo we can clean up message strings and replace 'Windows PowerShell' with 'PowerShell' in resx files.

## Fix

Find and replace 'Windows PowerShell' with 'PowerShell' in .resx files. Then revert some replacements (related workflows, "Windows PowerShell 3.0 and higher" and so on).

## Additional considerations

The PR doesn't address *.cs files (which have 'Windows PowerShell' strings in comments).

### CODE REVIEW

**Comment by SteveL-MSFT** [comment]
I think we should replace `Windows PowerShell` with just `PowerShell` and not `PowerShell Core`

**Comment by dantraMSFT** [comment]
@SteveL-MSFT: I agree; Core identifies an underlying technology; not PowerShell itself.

**Comment by iSazonov** [comment]
I agree. Will do.

File: src/Microsoft.PowerShell.Commands.Utility/resources/Debugger.resx [hunk 1]
```diff
@@ -142,7 +142,7 @@
     <value>You cannot debug the default host Runspace using this cmdlet. To debug the default Runspace use the normal debugging commands from the host.</value>
   </data>
   <data name="RunspaceDebuggingNoHostRunspaceOrDebugger" xml:space="preserve">
-    <value>Cannot debug Runspace. The host has no debugger. Try debugging the Runspace inside the Windows PowerShell console or the Windows PowerShell ISE, both of which have built-in debuggers.</value>
+    <value>Cannot debug Runspace. The host has no debugger. Try debugging the Runspace inside the PowerShell Core console or the Visual Studio Code, both of which have built-in debuggers.</value>
```
Comment (dantraMSFT) [inline src/Microsoft.PowerShell.Commands.Utility/resources/Debugger.resx]: >or the [](start = 122, length = 6)

Suggestion: 'or the Visual Studio Code' is rather cumbersome to read IMHO.
How about 'or with Visual Studio Code.'

Comment (iSazonov) [reply src/Microsoft.PowerShell.Commands.Utility/resources/Debugger.resx]: Fixed.

File: src/Microsoft.PowerShell.Commands.Utility/resources/NewObjectStrings.resx [hunk 2]
```diff
@@ -142,6 +142,6 @@
     <value>Cannot create type. Only core types are supported in this language mode.</value>
   </data>
   <data name="ApartmentNotSupported" xml:space="preserve">
-    <value>{0} Please note that Single-Threaded Apartment is not supported in PowerShell Core.</value>
+    <value>{0} Please note that Single-Threaded Apartment is not supported in PowerShell.</value>
```
Comment (SteveL-MSFT) [inline src/Microsoft.PowerShell.Commands.Utility/resources/NewObjectStrings.resx]: We should leave this one since it's Core specific (until I get my PR submitted)

File: src/System.Management.Automation/resources/HostInterfaceExceptionsStrings.resx [hunk 3]
```diff
@@ -121,7 +121,7 @@
     <value>An error of type "{0}" has occurred.</value>
   </data>
   <data name="HostFunctionNotImplemented" xml:space="preserve">
-    <value>A command that prompts the user failed because the host program or the command type does not support user interaction. Try a host program that supports user interaction, such as the Windows PowerShell Console or Windows PowerShell ISE, and remove prompt-related commands from command types that do not support user interaction, such as Windows PowerShell workflows.</value>
+    <value>A command that prompts the user failed because the host program or the command type does not support user interaction. Try a host program that supports user interaction, such as the PowerShell Console, and remove prompt-related commands from command types that do not support user interaction, such as PowerShell workflows.</value>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/resources/HostInterfaceExceptionsStrings.resx]: Remove mention of workflows which isn't supported

File: src/System.Management.Automation/resources/Modules.resx [hunk 4]
```diff
@@ -625,7 +625,7 @@
     <value>The specified module '{0}' was not found. Update the Name parameter to point to a valid path, and then try again. </value>
   </data>
   <data name="WorkflowModuleNotSupportedInPowerShellCore" xml:space="preserve">
-    <value>Cannot load the workflow module '{0}'. Workflow is not supported in PowerShell Core.</value>
+    <value>Cannot load the workflow module '{0}'. Workflow is not supported in PowerShell.</value>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/resources/Modules.resx]: Put Core back as this is specific to Core

File: src/System.Management.Automation/resources/MshSnapInCmdletResources.resx [hunk 5]
```diff
@@ -121,15 +121,15 @@
...
-    <value>The format of the specified snap-in name was not valid.  Windows PowerShell snap-in names can only contain alpha-numeric characters, dashes, underscores and periods. Correct the name and then try the operation again.</value>
+    <value>The format of the specified snap-in name was not valid.  PowerShell snap-in names can only contain alpha-numeric characters, dashes, underscores and periods. Correct the name and then try the operation again.</value>
   </data>
   <data name="LoadSystemSnapinAsModule" xml:space="preserve">
-    <value>Cannot add Windows PowerShell snap-in {0} because it is a system Windows PowerShell module. Use Import-Module to load the module.</value>
+    <value>Cannot add PowerShell snap-in {0} because it is a system PowerShell module. Use Import-Module to load the module.</value>
   </data>
   <data name="CustomPSSnapInNotSupportedInPowerShellCore" xml:space="preserve">
-    <value>The custom Windows PowerShell snap-in is not supported in PowerShell Core.</value>
+    <value>The custom PowerShell snap-in is not supported in PowerShell.</value>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/resources/MshSnapInCmdletResources.resx]: Put Core back as this is specific to Core not supporting PSSnapin

File: src/System.Management.Automation/resources/MshSnapinInfo.resx [hunk 6]
```diff
@@ -118,19 +118,19 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="MonadRootRegistryAccessFailed" xml:space="preserve">
-    <value>Unable to access Windows PowerShell registry information.</value>
+    <value>Unable to access PowerShell registry information.</value>
   </data>
   <data name="MonadEngineRegistryAccessFailed" xml:space="preserve">
-    <value>Unable to access Windows PowerShell PowerShellEngine registry information.</value>
+    <value>Unable to access PowerShell PowerShellEngine registry information.</value>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/resources/MshSnapinInfo.resx]: Seems redundant, perhaps just: PowerShell Engine registry

File: src/System.Management.Automation/resources/ParserStrings.resx [hunk 7]
```diff
@@ -1192,7 +1192,7 @@ ModuleVersion : Version of module to import. If used, ModuleName must represent
     <value>Exception occurred when post-parsing dynamic keyword '{0}' with details '{1}'.</value>
   </data>
   <data name="WorkflowNotSupportedInPowerShellCore" xml:space="preserve">
-    <value>Workflow is not supported in PowerShell Core.</value>
+    <value>Workflow is not supported in PowerShell.</value>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/resources/ParserStrings.resx]: Put Core back

File: src/System.Management.Automation/resources/ParserStrings.resx [hunk 8]
```diff
@@ -1261,7 +1261,7 @@ ModuleVersion : Version of module to import. If used, ModuleName must represent
     <value>Variant.GetAccessor cannot handle {0}.</value>
   </data>
   <data name="ConfigurationNotSupportedInPowerShellCore" xml:space="preserve">
-    <value>Configuration keyword is not supported in PowerShell Core.</value>
+    <value>Configuration keyword is not supported in PowerShell.</value>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/resources/ParserStrings.resx]: Put Core back

File: src/System.Management.Automation/resources/ParserStrings.resx [hunk 9]
```diff
@@ -1355,7 +1355,7 @@ ModuleVersion : Version of module to import. If used, ModuleName must represent
 '{1}'.</value>
   </data>
   <data name="CantActivateDocumentInPowerShellCore" xml:space="preserve">
-    <value>Cannot run a document in PowerShell Core: {0}.</value>
+    <value>Cannot run a document in PowerShell: {0}.</value>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/resources/ParserStrings.resx]: I thought we fixed this so this error should no longer come up?  If it's still applicable, put back Core

File: src/System.Management.Automation/resources/RemotingErrorIdStrings.resx [hunk 10]
```diff
@@ -654,13 +654,13 @@
     <value>Session configuration "{0}" was not found.</value>
   </data>
   <data name="CSCmdsShellNotPowerShellBased" xml:space="preserve">
-    <value>Session configuration "{0}" is not a Windows PowerShell-based shell.</value>
+    <value>Session configuration "{0}" is not a PowerShell-based shell.</value>
 </data>
   <data name="CSCmdsPowerShellCoreShellNotModifiable" xml:space="preserve">
-    <value>Session configuration "{0}" is a PowerShell Core-based shell. Please use PowerShell Core to modify it.</value>
+    <value>Session configuration "{0}" is a PowerShell-based shell. Please use PowerShell to modify it.</value>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/resources/RemotingErrorIdStrings.resx]: This error seems Core specific, please revert

File: src/System.Management.Automation/resources/RemotingErrorIdStrings.resx [hunk 11]
```diff
@@ -654,13 +654,13 @@
...
   </data>
   <data name="CSCmdsWindowsPowerShellCoreNotModifiable" xml:space="preserve">
-    <value>Session configuration "{0}" is a Windows PowerShell-based shell. Please use Windows PowerShell to modify it.</value>
+    <value>Session configuration "{0}" is a PowerShell-based shell. Please use PowerShell to modify it.</value>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/resources/RemotingErrorIdStrings.resx]: This error seems Windows PS specific, please revert

File: src/System.Management.Automation/resources/RemotingErrorIdStrings.resx [hunk 12]
```diff
@@ -952,7 +952,7 @@ All WinRM sessions connected to Windows PowerShell session configurations, such
     <value>A running command could not be found for this PSSession.</value>
   </data>
   <data name="NetFrameWorkV2NotInstalled" xml:space="preserve">
-    <value>The Microsoft .NET Framework 2.0, which is required for Windows PowerShell 2.0, is not installed. Install the .NET Framework 2.0 and retry.</value>
+    <value>The Microsoft .NET Framework 2.0, which is required for PowerShell 2.0, is not installed. Install the .NET Framework 2.0 and retry.</value>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/resources/RemotingErrorIdStrings.resx]: Windows PS specific, please revert although PS2.0 is deprecated, so maybe this error and code should be removed in another PR

File: src/System.Management.Automation/resources/RemotingErrorIdStrings.resx [hunk 13]
```diff
@@ -1646,13 +1646,13 @@ All WinRM sessions connected to Windows PowerShell session configurations, such
     <value>The SSH transport process has abruptly terminated causing this remote session to break.</value>
   </data>
   <data name="InvalidProcessorArchitecture" xml:space="preserve">
-    <value>PowerShell Core does not support WOW64. The binary must match the architecture of the processor.</value>
+    <value>PowerShell does not support WOW64. The binary must match the architecture of the processor.</value>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/resources/RemotingErrorIdStrings.resx]: Core specific, please revert

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: src/Microsoft.PowerShell.Commands.Utility/resources/NewObjectStrings.resx (hunk 2 above)
Comment (iSazonov) [reply src/Microsoft.PowerShell.Commands.Utility/resources/NewObjectStrings.resx]: Fixed.

File: src/System.Management.Automation/resources/HostInterfaceExceptionsStrings.resx (hunk 3 above)
Comment (iSazonov) [reply src/System.Management.Automation/resources/HostInterfaceExceptionsStrings.resx]: Fixed.

File: src/System.Management.Automation/resources/Modules.resx (hunk 4 above)
Comment (iSazonov) [reply src/System.Management.Automation/resources/Modules.resx]: Fixed.

File: src/System.Management.Automation/resources/MshSnapInCmdletResources.resx (hunk 5 above)
Comment (iSazonov) [reply src/System.Management.Automation/resources/MshSnapInCmdletResources.resx]: Fixed.

File: src/System.Management.Automation/resources/MshSnapinInfo.resx (hunk 6 above)
Comment (iSazonov) [reply src/System.Management.Automation/resources/MshSnapinInfo.resx]: Fixed.

File: src/System.Management.Automation/resources/ParserStrings.resx (hunk 7 above)
Comment (iSazonov) [reply src/System.Management.Automation/resources/ParserStrings.resx]: Fixed.

File: src/System.Management.Automation/resources/ParserStrings.resx (hunk 8 above)
Comment (iSazonov) [reply src/System.Management.Automation/resources/ParserStrings.resx]: Fixed.

File: src/System.Management.Automation/resources/ParserStrings.resx (hunk 9 above)
Comment (iSazonov) [reply src/System.Management.Automation/resources/ParserStrings.resx]: I don't know about applicable but `CantActivateDocumentInPowerShellCore` is still used in `NativeCommandProcessor.cs ` for Nano and IoT.

Reverted.

File: src/System.Management.Automation/resources/RemotingErrorIdStrings.resx (hunk 10 above)
Comment (iSazonov) [reply src/System.Management.Automation/resources/RemotingErrorIdStrings.resx]: Fixed.

File: src/System.Management.Automation/resources/RemotingErrorIdStrings.resx (hunk 11 above)
Comment (iSazonov) [reply src/System.Management.Automation/resources/RemotingErrorIdStrings.resx]: Fixed.

File: src/System.Management.Automation/resources/RemotingErrorIdStrings.resx (hunk 12 above)
Comment (iSazonov) [reply src/System.Management.Automation/resources/RemotingErrorIdStrings.resx]: Fixed.

File: src/System.Management.Automation/resources/RemotingErrorIdStrings.resx (hunk 13 above)
Comment (iSazonov) [reply src/System.Management.Automation/resources/RemotingErrorIdStrings.resx]: Fixed.

**Review by SteveL-MSFT — APPROVED** [review]
LGTM

**Comment by iSazonov** [comment]
@SteveL-MSFT Thanks for fast review!
